src/objects/zcl_abapgit_object_iwmo.clas.abap

```diff
@@ -90,7 +90,37 @@ CLASS ZCL_ABAPGIT_OBJECT_IWMO IMPLEMENTATION.
 
   METHOD zif_abapgit_object~jump.
 
-    zcx_abapgit_exception=>raise( |TODO: Jump| ).
+    DATA: lv_mdl_technical_name TYPE /iwbep/med_mdl_technical_name,
+          lv_version            TYPE /iwbep/med_mdl_version,
```

Comment:
Do these structures in systems lower 740 without embedded gateway exist?

Reply:
That might be a problem yes.

Reply:
strictly they are not needed? just move the value directly into `fval`? with offset and length fixes